python-client/collect-stats.py

```diff
@@ -0,0 +1,178 @@
+import requests
+import json
+import asyncio
+import os
+
+# BASEURL = 'https://qpp.cms.gov/api/submissions/web-interface/'
+LOCAL_BASEURL = 'http://localhost:3000/api/submissions/web-interface/'
+LIMIT = 100
+
+class CollectStats:
+    def get_url(self, path):
+        return LOCAL_BASEURL+path
+
+    async def do_main_task(self):
```

Comment:
why is this async?  I get the intent, but I don't see any awaits in the code.

Reply:
yeah @jpattersonz. I thought I can use it like javascript, but the syntax is a little different here. I will update this. thanks